docs/psmysql-pro.md

```diff
@@ -12,6 +12,7 @@ Find the list of capabilities available in Percona Server for MySQL Pro:
 | ----------------------------------- | ------------- | -------------|
 | [FIPS compliance](fips.md)| 8.0.35-27 | Enables all commercial cloud service providers who want to sell and expand their footprint for US government entities. |
 | [FIPS compliance](fips.md)| 8.0.36-28 | The FIPS feature has been tested on Percona Server for MySQL Pro 8.0.36-28. There are no other changes to this release.|
+| [FIPS compliance](fips.md)| 8.0.37-29 | The FIPS feature has been tested on Percona Server for MySQL Pro 8.0.37-29. There are no other changes to this release.|
```

Comment:
There are no changes to this release
The "other" is redundant.

Reply:
The FIPS feature has been tested on Percona Server for MySQL Pro 8.0.37-29. There are no other changes to this release.

I still think the "other" is not needed. Testing does not change the software.

Reply:
I've fixed it already both for 8.0.36-28 and 8.0.37-29.